cmd/report.go

```diff
@@ -0,0 +1,100 @@
+package cmd
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"net/http"
+	"runtime"
+	"strings"
+
+	"go.k6.io/k6/execution"
+	"go.k6.io/k6/lib/consts"
+)
+
+type report struct {
+	Version    string         `json:"k6_version"`
+	Executors  map[string]int `json:"executors"`
+	VUsMax     int64          `json:"vus_max"`
+	Iterations uint64         `json:"iterations"`
+	Duration   string         `json:"duration"`
+	GoOS       string         `json:"goos"`
+	GoArch     string         `json:"goarch"`
+	Modules    []string       `json:"modules"`
+	Outputs    []string       `json:"outputs"`
+}
+
+func createReport(execScheduler *execution.Scheduler, importedModules []string, outputs []string) report {
+	executors := make(map[string]int)
+	for _, ec := range execScheduler.GetExecutorConfigs() {
+		executors[ec.GetType()]++
+	}
+
+	// collect the report only with k6 public modules
+	publicModules := make([]string, 0, len(importedModules))
+	for _, module := range importedModules {
+		// Exclude JS modules extensions to prevent to leak
+		// any user's custom extensions
+		if strings.HasPrefix(module, "k6/x") {
```

Comment:
Would make sense to keep a list, even if manually, here to also get usage stats for the extensions that we do own? At least until we find a better way to do so.

From the product perspective, I guess it could make lot of sense to know what extensions worth some attention and which ones not.

Reply:
```
xk6-client-tracing
xk6-notification
xk6-disruptor
xk6-sql
xk6-exec
xk6-kubernetes
xk6-ssh
xk6-loki
xk6-dashboard
xk6-output-influxdb
xk6-output-kafka
xk6-output-timescaledb
```
This is the list of extensions marked as official here: https://k6.io/docs/extensions/get-started/explore. None of them sounds so relevant at the moment to justify the risk of having this logic. Let me know if you have a different opinion.

Reply:
Sounds fair! Thanks!